gapis/api/transform2/transform_chain.go

```diff
@@ -185,7 +187,7 @@ func (chain *TransformChain) GetNextTransformedCommands(ctx context.Context) err
 		log.E(ctx, "Replay error (%v:%v): %v", chain.currentCommandID, currentCommand, err)
 	}
 
-	chain.currentCommandID++
+	chain.currentCommandID.id++
```

Comment:
I think we should have an id.NextAfter() that returns a new ID and errors if you're trying to NextAfter on a Begin or End.

Reply:
I am not very sure about making the commandID struct too clever but I will take a look at this.

Reply:
I just renamed NextAfter to Increment